### ISSUE TITLE
expose serve-d settings in package settings

### ISSUE THREAD

**Issue by WebFreak001** [Pure-D/atom-ide-d#45]
The serve-d settings (LSP per project or global configuration) should be available to edit in the Atom settings & possibly also per-project if Atom supports it.

**Comment by aminya** [comment]
Yes, I was thinking about this. There are two ways to approach this: manually write the config files, or automatically. I want to see if there is any way to do this in an automatic way. 

I need to check if the automatic method is supported, and if not trying to implement it in `atom-languageclient` or `atom-vscode`. 

https://github.com/atom-community/atom-languageclient
https://github.com/atom-community/atom-vscode
